src/components/common/CommonButtons/CommonButtons.style.js

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components';
+import { bold24, m_bold24 } from '../../../styles/font';
+
+export const S = {};
+
+S.JoinUsContainer = styled.div`
```

Comment:
```suggestion
S.JoinUsContainer = styled.button`
```
버튼이니까 버튼 태그 사용하면 좋을 거 같아요!